src/Runner.Common/Terminal.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
```

Comment:
I do not quite get why we need to change this file.

Reply:
Undid the renaming, makes more sense to leave it `Silent`

Reply:
`Silent = StringUtil.ConvertToBoolean(Environment.GetEnvironmentVariable(Constants.Variables.Agent.PrintLogToStdout));`

This is needed to avoid double printing, should we set it somewhere else?